network/unix_domain_server_test.cc

```diff
@@ -0,0 +1,190 @@
+/*
+Copyright 2025 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include "unix_domain_server.h"
+
+#include <gtest/gtest.h>
+
+namespace Network
+{
+namespace
+{
+
+const std::string kTestServerAddress = "test_server_socket";
```

Comment:
Avoid static variables with non-trivial constructors and destructors. For strings, prefer to use `constexpr std::string_view` or `constexpr char[]`
```suggestion
constexpr std::string_view kTestServerAddress = "test_server_socket";
```

The order of initialization and destruction is not well defined which could lead to subtle bugs: accessing a variable before it has been initialized, or after it has been destroyed. See the [Google C++ Style guide](https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables) and [Tip of the Week #140: Constants: Safe Idioms](https://abseil.io/tips/140)